### PR TITLE
Underdogs_Trek.com Login,  Marlin Refactor

### PR DESCRIPTION
**Pull request by ArtuomKudryashov**

### CODE REVIEW

File: src/test/java/school/redrover/GroupUnderdogsTest.java [hunk 1]
```diff
@@ -14,8 +14,9 @@ public class GroupUnderdogsTest {
     WebDriver driver;
 
     String userName = "academic198405@gmail.com";
-    String password = "StateOfLiberty2021!";
-    String wrongpassword = "StateOfLiberty2021!2";
+    String password = "BikeTrekMarlyn4!";
+    String wrongpassword = "Sbbhbhblnk!2";
```
Comment (taysan003) [inline src/test/java/school/redrover/GroupUnderdogsTest.java]: Use camel case like wrongPassword

File: src/test/java/school/redrover/GroupUnderdogsTest.java [hunk 2]
```diff
@@ -98,6 +99,7 @@ public void artuomTrack_correct_CredentialTest() throws InterruptedException {
         WebElement passwordField = driver.findElement(By.xpath("//*[@id=\"j_password\"]"));
```
Comment (taysan003) [inline src/test/java/school/redrover/GroupUnderdogsTest.java:99]: better to use By.id("j_password\")

File: src/test/java/school/redrover/GroupUnderdogsTest.java [hunk 3]
```diff
@@ -14,8 +14,9 @@ public class GroupUnderdogsTest {
     WebDriver driver;
 
     String userName = "academic198405@gmail.com";
-    String password = "StateOfLiberty2021!";
-    String wrongpassword = "StateOfLiberty2021!2";
+    String password = "BikeTrekMarlyn4!";
+    String wrongpassword = "Sbbhbhbln2";
```
Comment (val-kan) [inline src/test/java/school/redrover/GroupUnderdogsTest.java:18]: wrongPassword

Comment (ArtuomKudryashov) [reply src/test/java/school/redrover/GroupUnderdogsTest.java:18]: Да согласен

Comment (ArtuomKudryashov) [reply src/test/java/school/redrover/GroupUnderdogsTest.java]: Исправил

File: src/test/java/school/redrover/GroupUnderdogsTest.java [hunk 4]
```diff
@@ -79,7 +80,7 @@ public void maksinTestInactive() {
     @Test
     public void artuomTrack_correct_CredentialTest() throws InterruptedException {
```
Comment (val-kan) [inline src/test/java/school/redrover/GroupUnderdogsTest.java:81]: question - why did you use mixed (Camel+Snake) cases in test names? Is it possible to name it like : artuomTrackCorrectCredentialTest(); ?

Comment (ArtuomKudryashov) [reply src/test/java/school/redrover/GroupUnderdogsTest.java:81]: Which  name? Please give me a few more details

Comment (ArtuomKudryashov) [reply src/test/java/school/redrover/GroupUnderdogsTest.java:81]: I have found what you mean

**Review by maksin82 — APPROVED** [review]

**Review by JuliaAfa — APPROVED** [review]

**Pull request merged**